src/server/views/help/index.html

```diff
@@ -0,0 +1,48 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Get help with your form
+{% endblock %}
+
+{% block content %}
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {% if form %}
+        <h1 class="govuk-heading-l">Get help with this form</h1>
+      {% endif %}
+
+      {# Example pattern. Uncomment when we have some real form help content
+      {% switch form.id %}
```

Comment:
We could replace this with something like:

`{% include "partials/help/{{form.id}}.html" ignore missing %}` 

And in the directory we would include the form-specific help partials e.g. `partials/help/66684c0e128c09b3a8444ce8.html`

[ignore missing](https://mozilla.github.io/nunjucks/templating#include) would protect us from missing content.

Reply:
Resolved with d74a8b6f5e08c158f0277a96f3a712ca562d5830